### ISSUE TITLE
Published under new package name

### ISSUE THREAD

**Issue by chadselph** [guardrail-dev/guardrail-gradle-plugin#154]
It doesn't seem like the gradle plugin has been published since the move from com.twilio to dev.guardrail

**Comment by blast-hardcheese** [comment]
Yeah, I registered a new repo for it in the Gradle plugin repo, but I haven't had time to get back to it. If this is blocking you I can try to get to it this week

**Comment by blast-hardcheese** [comment]
Sorry for this taking so long Chad, I haven't had a tremendous amount of recreational programming time recently. I haven't forgotten about this.

**Comment by chadselph** [comment]
No worries, I'm fine with the older version under the Twilio namespace.

**Comment by blast-hardcheese** [comment]
Alright. v0.71.0 published, pending approval:

<img width="786" alt="image" src="https://github.com/guardrail-dev/guardrail-gradle-plugin/assets/278900/389f9da2-03bc-468d-a951-3bc0e221cb7f">
